diff -Naur llvm/projects/compiler-rt/lib/builtins/gcc_personality_v0.c llvm-fixed/projects/compiler-rt/lib/builtins/gcc_personality_v0.c
--- llvm/projects/compiler-rt/lib/builtins/gcc_personality_v0.c	2015-09-03 06:31:52.241190187 -0400
+++ llvm-fixed/projects/compiler-rt/lib/builtins/gcc_personality_v0.c	2015-09-03 06:22:26.152300468 -0400
@@ -159,7 +159,7 @@
         return _URC_CONTINUE_UNWIND;
         
     /* There is nothing to do if there is no LSDA for this frame. */
-    const uint8_t* lsda = _Unwind_GetLanguageSpecificData(context);
+    const uint8_t* lsda = (const uint8_t*) _Unwind_GetLanguageSpecificData(context);
     if ( lsda == (uint8_t*) 0 )
         return _URC_CONTINUE_UNWIND;
 
